scripts/sct_run_batch.py

```diff
@@ -398,6 +397,10 @@ def reset_streams():
     else:
         jobs = args.jobs
 
```

Comment:
should there be a \n before the RUNNING?

Reply:
There's one after the origin report above, so it looks right to me.